src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -593,31 +692,16 @@ private void EnterParameter(ParameterSymbol parameter)
                 Debug.Assert(!IsConditionalState);
                 if (slot > 0 && parameter.RefKind != RefKind.Out)
                 {
-                    TypeSymbolWithAnnotations paramType = parameter.Type;
-
-                    if (paramType.IsReferenceType)
-                    {
-                        if (paramType.IsNullable != false)
-                        {
-                            if (slot >= this.State.Capacity) Normalize(ref this.State);
-
-                            this.State[slot] = (paramType.IsNullable == true) ? (bool?)false : null;
```

Comment:
>this.State[slot] = (paramType.IsNullable == true) ? (bool?)false : null; [](start = 28, length = 72)

It Is not obvious why this is no longer needed. Is this handled by Normalize?

Reply:
Yes, `Normalize` will set the state of `this.State[slot]`, and the state of any fields or properties will be set lazily, when the field or property value is set.